fleet/lib/k8up-post/overlays/ruka/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+resources:
+  - backup-namespace.yaml
```

Comment:
perhaps it would be good to call it backup-daily instead of backup-namespace, so we can group all namespaces that will be backed up every day. If we need to change the schedule we can create another one. (ie backup-weekly)